### ISSUE TITLE
"Upload incomplete" after upgrade

### ISSUE THREAD

**Issue by evenbrenden** [jotta/jotta-cli-issues#131]
Make sure you are running the latest version of jotta-cli before reporting an issue.

**jotta-cli release** (`jotta-cli version`): 0.9.38023

**Description of problem:**

After upgrading from 0.7.35160 to 0.9.38023, most (but not all) new files that I try to back up are not uploaded. I cannot see any pattern in the attributes for the failing files, except that they are in the 10MB range or larger (small text files appears to always work). If I remove the files from the backup folder, the logs show that they are deleted, so it seems that they are registered somehow. I tried clearing `~/.jottad/` and did a full reupload (with a new machine ID), which was successful up to all files backed up prior to the upgrade, but most subsequent uploads fail.

**Expected:**

All files are uploaded.

**jotta-cli status (`jotta-cli status`):**
```
------------------------------------------------------------------------------
 Account   : evenbrenden@gmail.com on Jottacloud
 Usage     : 3.38TiB / ( Unlimited )
 Device    : home
------------------------------------------------------------------------------
 Backups   :
------------------------------------------------------------------------------
   Path      : /media/ted15
   Files     : 98684 files / 1.14TiB
   Status    : Up to date!

------------------------------------------------------------------------------
OK
```

**Relevant logs for the issue (`cat ~/.jottad/jottabackup.log`)**
```
pid:26386 2021/01/27 19:46:29 Upload incomplete
```

**Traceback**
```

```

**Additional info:**

**Comment by evenbrenden** [comment]
Worth mentioning that after the upgrade, I starting out trying to uploading a large file (5GB), which failed. But as mentioned in the description, files in the 10MB range fail too.

**Comment by Kimbsen** [comment]
This sounds disconcerting. I am not able to reproduce that with either 0.7 or 0.9 so i'm not sure what is going on. Could you please send the logfile to support@jottacloud.com and we can continue the conversation there?

**Comment by Kimbsen** [comment]
Right. We figured out what was wrong and now we have fixed it.

What happened was that the incomplete file error was not treated as an error causing the local backup state to be corrupt and the file to not be uploaded again even though the server did not have the entire file. We fixed that issue in version 0.9.39279 released january 28.

We just released version 0.9.39536 which cleans up the corrupt local backup state ensuring that the incomplete files are uploaded.

Thanks for reporting this!
